common_scripts/enigma_docker_common/provider.py

```diff
@@ -40,6 +42,14 @@ def __init__(self, config: dict):
             self._enigma_contract_abi_filename = config.get('ENIGMA_CONTRACT_ABI_FILENAME', 'Enigma.json')
             self._enigma_contract_abi_filename_zip = config.get('ENIGMA_CONTRACT_ABI_FILENAME_ZIPPED', 'Enigma.zip')
 
+        self._voting_abi_directory = config.get('VOTING_CONTRACT_ABI_DIRECTORY', 'voting')
+        self._voting_abi_filename = config.get('VOTING_CONTRACT_ABI_FILENAME', 'VotingETH.json')
+        self._voting_abi_filename_zip = config.get('VOTING_CONTRACT_ABI_FILENAME_ZIPPED', 'VotingETH.zip')
+
+        self._sample_abi_directory = config.get('SAMPLE_CONTRACT_ABI_DIRECTORY', 'sample')
+        self._sample_abi_filename = config.get('SAMPLE_CONTRACT_ABI_FILENAME', 'Sample.json')
```

Comment:
Same as above

Reply:
Same as above ;)

Reply:
removed in 1cb172a817f7a75b378d4b04378a66f3ca7c4ecc